UI_UX/Frontend/src/styles/Achievements.scss

```diff
@@ -0,0 +1,80 @@
+.achievements {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  padding-top: 70px;
+  padding-bottom: 70px;
+  padding-left: 75px;
+  overflow-y: scroll;
+
+  &> li {
+    width: 150px;
+    height: 150px;
+    transform: rotate(45deg);
+    background-image: linear-gradient(85deg, #001358, #125ac4);
```

Comment:
We'll eventually move this to a `colors.scss` but it's fine for the time being. Same for spacings.